Libplanet.Stun/Libplanet.Stun.csproj

```
@@ -1,4 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Libplanet.Stun</PackageId>
+    <Title>Libplanet.Stun</Title>
+    <Summary>A STUN/TURN server implementation for Libplanet.</Summary>
+    <Description>A STUN/TURN server implementation for Libplanet.</Description>
```

Actually, `Libplanet.Stun` contains only client implementation. it needs STUN/TURN server like [coturn](https://github.com/coturn/coturn). 😅 

```suggestion
    <Summary>A STUN/TURN client implementation for Libplanet.</Summary>
    <Description>A STUN/TURN client implementation for Libplanet.</Description>
```

---

Hmm... guess I should've referred to the RFC more carefully.